schema/4.0.0/schema.md

```diff
@@ -1112,6 +1112,72 @@ See also `default_embedding` in `uns`.
 </tbody></table>
 <br>
 
+
+### {column}_colors
+
+<table><tbody>
+  <tr>
+    <th>Key</th>
+      <td>
+        {column}_colors where {column} MUST be the name of a <code>category</code> data type column in <code>obs</code> that<br> is annotated by the data submitter or curator. The following columns that are annotated by CELLxGENE<br> Discover MUST NOT be specified as {category}:<br><br>
```

Comment:
Discover MUST NOT be specified as {~category~column}:

Reply:
DOH. You'd think that global replace had not been invented.

Addressed in 0f62f4ed2625740ab6cbfc35d6f32123fd2716d4.